custom-coreos-disk-images.sh

```diff
@@ -137,7 +141,15 @@ main() {
     # Default Metal Image Size
     metal_image_size="${metal_image_size:-3072}"
 
-    # Make a local tmpdir and outidr
+    # Default kernel arguments are different for FCOS/RHCOS
+    if [ -z "${extra_kargs:-}" ]; then
+        extra_kargs="" # RHCOS
+        if [ "$osname" == 'fedora-coreos' ]; then
+            extra_kargs="mitigations=auto,nosmt" # FCOS
```

Comment:
This one is clearly an example of what should be a managed karg.

I guess technically we should be appending to this, not overriding no?

Reply:
No. For this particular use case podman folks don't want `mitigations=auto,nosmt`.

Reply:
FWIW, the images we make for podman do not include those ...

```
ccore@localhost:~$ cat /proc/cmdline 
BOOT_IMAGE=(hd0,gpt3)/ostree/fedora-coreos-72463aff70776820eade261d86d45fad4eb96f291c507b3f4319c361c58f6d52/vmlinuz-6.11.3-200.fc40.x86_64 rw ignition.firstboot ostree=/ostree/boot.1/fedora-coreos/72463aff70776820eade261d86d45fad4eb96f291c507b3f4319c361c58f6d52/0 console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu
```

Reply:
right, because you've been using these scripts to build the images which haven't been including `mitigations=auto,nosmt`

Reply:
I'll merge and we can make any more changes we want in a followup.